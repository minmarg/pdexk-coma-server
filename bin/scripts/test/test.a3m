>ss_pred
CCCCCCCCCCCCCCCCCCCCCCCCCC
>ss_conf
95342431012222222223334579
>test
TESTXXXXXXXXXXXXXXXXXXXXXX
