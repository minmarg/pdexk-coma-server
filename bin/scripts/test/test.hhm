HHsearch 1.5
NAME  test
FAM   
FILE  test
COM   /home/minlag/software/hhsearch/hhmake -i /home/minlag/test_pdexkserver/bin/test/test.a3m -v 1 -cons -name test 
DATE  Tue Feb 23 10:52:22 2010
LENG  26 match states, 26 columns in multiple alignment
FILT  1 out of 3 sequences passed filter (-id 90 -cov 0 -qid 0 -qsc -20.00 -diff 100)
NEFF  1.0 
SEQ
>ss_pred
CCCCCCCCCCCCCCCCCCCCCCCCCC
>ss_conf
95342431012222222223334579
>Consensus
testxxxxxxxxxxxxxxxxxxxxxx
>test_consensus
TESTLLLLLLLLLLLLLLLLLLLLLL
>test
TESTXXXXXXXXXXXXXXXXXXXXXX
#
NULL   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	
HMM    A	C	D	E	F	G	H	I	K	L	M	N	P	Q	R	S	T	V	W	Y	
       M->M	M->I	M->D	I->M	I->I	D->M	D->D	Neff	Neff_I	Neff_D
       0	*	*	0	*	0	*	*	*	*	
T 1    *	*	*	*	*	*	*	*	*	*	*	*	*	*	*	*	0	*	*	*	1
       0	*	*	*	*	*	*	1000	0	0	

E 2    *	*	*	0	*	*	*	*	*	*	*	*	*	*	*	*	*	*	*	*	2
       0	*	*	*	*	*	*	1000	0	0	

S 3    *	*	*	*	*	*	*	*	*	*	*	*	*	*	*	0	*	*	*	*	3
       0	*	*	*	*	*	*	1000	0	0	

T 4    *	*	*	*	*	*	*	*	*	*	*	*	*	*	*	*	0	*	*	*	4
       0	*	*	*	*	*	*	1000	0	0	

L 5    3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	5
       *	*	*	*	*	*	*	1000	0	0	

L 6    3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	6
       *	*	*	*	*	*	*	1000	0	0	

L 7    3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	7
       *	*	*	*	*	*	*	1000	0	0	

L 8    3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	8
       *	*	*	*	*	*	*	1000	0	0	

L 9    3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	9
       *	*	*	*	*	*	*	1000	0	0	

L 10   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	10
       *	*	*	*	*	*	*	1000	0	0	

L 11   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	11
       *	*	*	*	*	*	*	1000	0	0	

L 12   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	12
       *	*	*	*	*	*	*	1000	0	0	

L 13   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	13
       *	*	*	*	*	*	*	1000	0	0	

L 14   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	14
       *	*	*	*	*	*	*	1000	0	0	

L 15   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	15
       *	*	*	*	*	*	*	1000	0	0	

L 16   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	16
       *	*	*	*	*	*	*	1000	0	0	

L 17   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	17
       *	*	*	*	*	*	*	1000	0	0	

L 18   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	18
       *	*	*	*	*	*	*	1000	0	0	

L 19   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	19
       *	*	*	*	*	*	*	1000	0	0	

L 20   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	20
       *	*	*	*	*	*	*	1000	0	0	

L 21   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	21
       *	*	*	*	*	*	*	1000	0	0	

L 22   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	22
       *	*	*	*	*	*	*	1000	0	0	

L 23   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	23
       *	*	*	*	*	*	*	1000	0	0	

L 24   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	24
       *	*	*	*	*	*	*	1000	0	0	

L 25   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	25
       *	*	*	*	*	*	*	1000	0	0	

L 26   3706	5728	4211	4064	4839	3729	4763	4308	4069	3323	5509	4640	4464	4937	4285	4423	3815	3783	6325	4665	26
       0	*	*	0	*	*	*	1000	0	0	

//
