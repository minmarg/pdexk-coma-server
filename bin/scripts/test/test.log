Building alignment for query with PSI-BLAST ...
Found 1 sequences in PSI-BLAST round 1 (E-value<1E-03, coverage>20%, db=nr90)
Found 1 sequences in PSI-BLAST round 2 (E-value<1E-03, coverage>20%, db=nr90)

Extending query sequence ...
Predicting secondary structure with PSIPRED ...
WARNING: did not find /home/minlag/software/dummydb.phr... using /usr/data/blast/db/nr70
Written alignment with 1 sequences to /home/minlag/test_pdexkserver/bin/test/test.a3m
.................................................. 1000 HMMs searched
.................................................. 2000 HMMs searched
.................................................. 3000 HMMs searched
.................................................. 4000 HMMs searched
.................................................. 5000 HMMs searched
.................................................. 6000 HMMs searched
.................................................. 7000 HMMs searched
.................................................. 8000 HMMs searched
.................................................. 9000 HMMs searched
.................................................. 10000 HMMs searched
.................................................. 11000 HMMs searched
.................................................. 12000 HMMs searched
.................................................. 13000 HMMs searched
......................................
Realigning 10 query-template alignments with maximum accuracy (MAC) algorithm ...
