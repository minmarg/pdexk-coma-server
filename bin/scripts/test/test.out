Query         test
Match_columns 26
No_of_seqs    1 out of 3
Neff          1.0 
Searched_HMMs 13767
Date          Tue Feb 23 10:52:34 2010
Command       /home/minlag/software/hhsearch/hhsearch -i /home/minlag/test_pdexkserver/bin/test/test.hhm -d /home/minlag/wastebasket/scop70_1.75_PFAM_PDB_PDEXK -v 1 -o /home/minlag/test_pdexkserver/bin/test/test.out -ssm 4 

 No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
  1 d1nbwa1 c.8.6.1 (A:92-256) Swi  46.7     2.3 0.00017   14.0   0.4    6    1-6       2-7   (165)
  2 d2d0oa1 c.8.6.1 (A:93-254) Dio  46.6     2.3 0.00017   14.0   0.4    5    1-5       2-6   (162)
  3 d1ibia2 g.39.1.3 (A:145-175) C  13.7      19  0.0014   11.8   0.1    2    3-4      12-13  (31)
  4 d2j4wd1 g.61.1.1 (D:421-454) A  12.3      24  0.0017   11.6   0.3    1    1-1       1-1   (34)
  5 d2j5la1 g.61.1.1 (A:479-512) A   9.9      32  0.0023   11.3   0.2    1    1-1       1-1   (34)
  6 d1gjja2 a.140.1.1 (A:111-153)    8.6      39  0.0028   11.1   0.2    1    1-1       1-1   (43)
  7 d4htci_ g.3.15.2 (I:) Hirudin    7.1      49  0.0035   10.9   0.2    1    1-1       1-1   (65)
  8 d1k44a_ d.58.6.1 (A:) Nucleosi   5.2      72  0.0052   10.5   0.2    1    1-1       1-1   (135)
  9 d2ffma1 d.267.1.1 (A:1-83) Hyp   4.8      82   0.006   10.3   0.2    1    4-4       9-9   (83)
 10 d1ixsa_ a.5.1.1 (A:) DNA helic   4.7      81  0.0059   10.3   0.2    1    1-1       1-1   (50)

No 1  
>d1nbwa1 c.8.6.1 (A:92-256) Swiveling domain of the glycerol dehydratase reactivase alpha subunit {Klebsiella pneumoniae [TaxId: 573]}
Probab=46.69   E-value=2.3  Score=13.98  Aligned_columns=6  Identities=67%

Q ss_pred             CCCCCC
Q ss_conf             953424
Q test_consensus    1 TESTLL    6 (26)
Q Consensus         1 test~~    6 (26)
                      ||||..
T Consensus         2 TEStMI    7 (165)
T d1nbwa1           2 TESTMI    7 (165)
T ss_dssp             ETTCEE
T ss_pred             CCCCCC
T ss_conf             543432


No 2  
>d2d0oa1 c.8.6.1 (A:93-254) Diol dehydratase-reactivating factor large subunit DdrA {Klebsiella oxytoca [TaxId: 571]}
Probab=46.59   E-value=2.3  Score=13.97  Aligned_columns=5  Identities=80%

Q ss_pred             CCCCC
Q ss_conf             95342
Q test_consensus    1 TESTL    5 (26)
Q Consensus         1 test~    5 (26)
                      ||||.
T Consensus         2 TeStm    6 (162)
T d2d0oa1           2 TESTM    6 (162)
T ss_dssp             CTTCE
T ss_pred             CCCCC
T ss_conf             54343


No 3  
>d1ibia2 g.39.1.3 (A:145-175) Cysteine-rich (intestinal) protein, CRP, CRIP {Japanese quail (Coturnix coturnix japonica), CRP2 [TaxId: 93934]}
Probab=13.70   E-value=19  Score=11.81  Aligned_columns=2  Identities=100%

Q ss_pred             CC
Q ss_conf             34
Q test_consensus    3 ST    4 (26)
Q Consensus         3 st    4 (26)
                      ||
T Consensus        12 st   13 (31)
T d1ibia2          12 ST   13 (31)
T ss_dssp             SS
T ss_pred             CC
T ss_conf             20


No 4  
>d2j4wd1 g.61.1.1 (D:421-454) Apical membrane antigen 1 {Malaria parasite (Plasmodium falciparum) [TaxId: 5833]}
Probab=12.31   E-value=24  Score=11.59  Aligned_columns=1  Identities=0%

Q ss_pred             C
Q ss_conf             9
Q test_consensus    1 T    1 (26)
Q Consensus         1 t    1 (26)
                      .
T Consensus         1 i    1 (34)
T d2j4wd1           1 I    1 (34)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


No 5  
>d2j5la1 g.61.1.1 (A:479-512) Apical membrane antigen 1 {Malaria parasite (Plasmodium falciparum) [TaxId: 5833]}
Probab=9.92    E-value=32  Score=11.30  Aligned_columns=1  Identities=0%

Q ss_pred             C
Q ss_conf             9
Q test_consensus    1 T    1 (26)
Q Consensus         1 t    1 (26)
                      .
T Consensus         1 I    1 (34)
T d2j5la1           1 I    1 (34)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


No 6  
>d1gjja2 a.140.1.1 (A:111-153) Thymopoietin, LAP2 {Human (Homo sapiens) [TaxId: 9606]}
Probab=8.58    E-value=39  Score=11.11  Aligned_columns=1  Identities=0%

Q ss_pred             C
Q ss_conf             9
Q test_consensus    1 T    1 (26)
Q Consensus         1 t    1 (26)
                      .
T Consensus         1 d    1 (43)
T d1gjja2           1 D    1 (43)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


No 7  
>d4htci_ g.3.15.2 (I:) Hirudin {Leech (Hirudo medicinalis) [TaxId: 6421]}
Probab=7.09    E-value=49  Score=10.87  Aligned_columns=1  Identities=0%

Q ss_pred             C
Q ss_conf             9
Q test_consensus    1 T    1 (26)
Q Consensus         1 t    1 (26)
                      .
T Consensus         1 v    1 (65)
T d4htci_           1 I    1 (65)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


No 8  
>d1k44a_ d.58.6.1 (A:) Nucleoside diphosphate kinase, NDK {Mycobacterium tuberculosis [TaxId: 1773]}
Probab=5.22    E-value=72  Score=10.47  Aligned_columns=1  Identities=100%

Q ss_pred             C
Q ss_conf             9
Q test_consensus    1 T    1 (26)
Q Consensus         1 t    1 (26)
                      |
T Consensus         1 t    1 (135)
T d1k44a_           1 T    1 (135)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


No 9  
>d2ffma1 d.267.1.1 (A:1-83) Hypothetical protein SAV1430 {Staphylococcus aureus [TaxId: 1280]}
Probab=4.78    E-value=82  Score=10.33  Aligned_columns=1  Identities=100%

Q ss_pred             C
Q ss_conf             4
Q test_consensus    4 T    4 (26)
Q Consensus         4 t    4 (26)
                      |
T Consensus         9 T    9 (83)
T d2ffma1           9 T    9 (83)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


No 10 
>d1ixsa_ a.5.1.1 (A:) DNA helicase RuvA subunit, C-terminal domain {Thermus thermophilus [TaxId: 274]}
Probab=4.75    E-value=81  Score=10.35  Aligned_columns=1  Identities=0%

Q ss_pred             C
Q ss_conf             9
Q test_consensus    1 T    1 (26)
Q Consensus         1 t    1 (26)
                      .
T Consensus         1 e    1 (50)
T d1ixsa_           1 E    1 (50)
T ss_dssp             C
T ss_pred             C
T ss_conf             9


Done!
